src/axom/inlet/Table.cpp

```diff
@@ -120,6 +120,13 @@ std::shared_ptr<Verifiable> Table::addStringArray(const std::string& name,
 std::shared_ptr<Table> Table::addGenericArray(const std::string& name,
                                               const std::string& description)
 {
+  if(m_sidreGroup->hasView("_inlet_array_indices"))
```

Comment:
These two string literals are used in quite a few spots at this point. Can you pull them into a common location?